patches/services-preferences-tracked-device_id_win.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/services/preferences/tracked/device_id_win.cc b/services/preferences/tracked/device_id_win.cc
+index 9dbd110674a9..038190311200 100644
+--- a/services/preferences/tracked/device_id_win.cc
++++ b/services/preferences/tracked/device_id_win.cc
+@@ -15,6 +15,9 @@
+ 
+ MachineIdStatus GetDeterministicMachineSpecificId(std::string* machine_id) {
+   DCHECK(machine_id);
++  if (IsMachineIdDisabled()) {
```

Comment:
same as above to make this one line. It's ok if it goes over 80 chars because the patches don't get linted anyway. We just want to reduce the potential for conflicts as much as possible